presto-orc/src/test/java/com/facebook/presto/orc/TestingColumnHandle.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
It's actually possible to reference it by adding a `test-jar` dependency:
```xml
<dependency>
    <groupId>com.facebook.presto</groupId>
    <artifactId>presto-spi</artifactId>
    <type>test-jar</type>
    <scope>test</scope>
</dependency>
```
But we don't do that for SPI anywhere else and I wouldn't do it here for one simple class.